CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * [FEATURE] Added sharding support to compactor when using the experimental TSDB blocks storage. #2113
 * [FEATURE] Add ability to override YAML config file settings using environment variables. #2147
   * `-config.expand-env`
+* [FEATURE] Add /config HTTP endpoint which exposes the current Cortex configuration as YAML.
```

Comment:
```suggestion
* [FEATURE] Add /config HTTP endpoint which exposes the current Cortex configuration as YAML. #2165
```

We now a script to check the changelog when we do a release, and it's based on the PR number in the changelog entry, reason why it's important to always add it.

Reply:
Good to know, thanks!